docker-compose.yml

```diff
@@ -1106,6 +1106,26 @@ services:
       - EX_APPS_NET=${EX_APPS_NET:-ipv4@localhost}
       - EX_APPS_COUNT=${EX_APPS_COUNT:-50}
 
+  appapi-harp:
+    image: ghcr.io/nextcloud/nextcloud-appapi-harp:latest
+    container_name: nextcloud-appapi-harp
+    network_mode: ${HP_NETWORK_MODE:-master_default}
```

Comment:
I'm curious about those parts, why do we need to set the network mode here actually?

Reply:
One can work with different kinds of configurations like host or any other network, for testing.